user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementMain.java

```diff
@@ -646,9 +717,26 @@ private static List<MeasurementRegistrationInformationNGS> mergeByPool(
     return finalMeasurements;
   }
 
-  private void displayRegistrationSuccessful() {
+
+  private void processRegistrationResults(int numberOfSuccesses, int numberOfRequests) {
+    if (numberOfSuccesses > 0 && numberOfSuccesses == numberOfRequests) {
+      // Only successful registrations
+      displayRegistrationSuccess(numberOfSuccesses);
+      return;
+    }
+    if (numberOfSuccesses > 0 && numberOfSuccesses < numberOfRequests) {
+      // We have successful registrations but also failures
+      displayRegistrationFailure(numberOfRequests -  numberOfSuccesses);
+      displayRegistrationSuccess(numberOfSuccesses);
+      return;
+    }
+    // There were only failing requests, none succeeded
+    displayRegistrationFailure(numberOfRequests -  numberOfSuccesses);
+  }
+
+  private void displayRegistrationSuccess(int numberOfSuccesses) {
     getUI().ifPresent(ui -> ui.access(() -> {
-      Toast toast = messageFactory.toast("measurement.registration.successful", new Object[]{},
+      Toast toast = messageFactory.toast("measurement.registration.successful", new Object[]{numberOfSuccesses},
```

Comment:
NIT: you can provide multiple parameters to the message. So it would be possible with little effort to show sth like `registered 5 out of 10 measurements` instead of `registered 5 measurements`. This would be more expressive.